docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
         - DJANGO_LOG_LEVEL=ERROR
         - LOG_LEVEL=INFO
         - EXPORT_DOWNLOAD_ROOT=/var/lib/eventkit/exports_download
-        - SITE_NAME
+        - SITE_NAME=cloud.eventkit.test
```

Comment:
Can you remove this change? I'm guessing this is just for your dev environment and we don't need it in master